staging/src/k8s.io/client-go/util/workqueue/delaying_queue.go

```diff
@@ -35,14 +35,17 @@ type DelayingInterface interface {
 
 // NewDelayingQueue constructs a new workqueue with delayed queuing ability
 func NewDelayingQueue() DelayingInterface {
-	return newDelayingQueue(clock.RealClock{}, "")
+	return NewDelayingQueueWithCustomClock(clock.RealClock{}, "")
```

Comment:
Please move the client-go changes to a separate PR.

Reply:
Done